grafeas-v1/README.md

```diff
@@ -0,0 +1,63 @@
+# Ruby Client for the Grafeas V1 API
+
+API Client library for the Grafeas V1 API
+
+The Grafeas API stores, and enables querying and retrieval of, critical metadata about all of your software artifacts.
+
+https://github.com/googleapis/google-cloud-ruby
+
+## Installation
+
+```
+$ gem install grafeas-v1
+```
+
+## Quick Start
+
+```ruby
+require "grafeas/v1"
+
+client = ::Grafeas::V1::Grafeas::Client.new
+request = my_create_request
```

Comment:
Should request be initialized to the definite request type?

Reply:
It's intentionally vague because this is generated with almost no context on what (semantically) makes up a reasonable request. I don't want people trying to copy-paste this code and expect that it will work unmodified.